guides/trailblazer/2.0/01-operation-basics.md

```diff
@@ -10,14 +10,14 @@ redirect_from: "/guides/trailblazer/2.0/01-getting-started-with-operation.html"
   {{ "app/blog_post/operation/create.rb:failure:../trailblazer-guides:operation-01" | tsnippet : "impl" }}
 
   ~~~6
-  The *Operation* is the central concept of the Trailblazer architecture. It is a simple service object that embraces and orchestrates all the business logic necessary to accomplish a certain task, such as creating a blog post, or updating a user.
+  The *Operation* is the central concept of the Trailblazer architecture. It's a simple service object that encapsulates and executes all the business logic necessary to accomplish a certain task, such as creating a blog post, or updating a user.
```

Comment:
I would actually like to keep "orchestrates" as "encapsulates" doesn't really make clear that it's happening somewhere else, not in this object!